gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/process/kubernetes/fetcher/ConfigMapEventFetcher.java

```diff
@@ -89,7 +89,8 @@ private Flowable<io.gravitee.kubernetes.client.model.v1.Event<ConfigMap>> watchC
                     .labelSelector(LabelSelector.equals(LABEL_GIO_TYPE, APIDEFINITIONS_TYPE))
                     .build()
             )
-            .retryWhen(errors -> errors.delay(RETRY_DELAY_MILLIS, TimeUnit.MILLISECONDS));
+            .retryWhen(errors -> errors.delay(RETRY_DELAY_MILLIS, TimeUnit.MILLISECONDS))
+            .repeat();
```

Comment:
Can you provide a unit test that reproduces the issue and so, that the fix does the job 🙏?